Update Docker image instructions (#3499)
snapcore/snapcraft: docker/README.md

# Creating docker containers for snapcraft

> By default the `Dockerfile` builds Ubuntu 16 (xenial) image with `snapcraft` from the `edge` channel.

Reviewer: ```suggestion
By default the `Dockerfile` builds Ubuntu 16.04 (Xenial) image with `snapcraft` from the `edge` channel.
```

    docker build . --no-cache

> It is however possible to choose base Ubuntu version and one of these channels (risk levels):

Reviewer: ```suggestion
It is however possible to choose the base Ubuntu version and the Snapcraft channel (risk levels):
```
